### ISSUE TITLE
[Accessibility] [Supporting the platform -GitHub]: Horizontal scroll bar appears when the page is zoomed to 400%. [Sev3][WCAG1.4.10]

### ISSUE THREAD

**Issue by Laxmidurga** [microsoft/tsdoc.org-website#34]
**Test Environment:**
OS Build: Windows 11
Version: 21H2 (OS Build 22000.318)
Browser: Edge
Version 96.0.1054.43 (Official build) (64-bit)
URL: https://tsdoc.org/
Screen reader: Narrator

**Prerequisites:** Set the Display resolution to 1280*1024 and browser zoom to 400%

**Repro Steps:**
1.	Open the above URL and login with valid credentials.
2.	TSDoc page will get displayed.
3.	Now, navigate to the ‘GitHub’ link and activate it.
4.	Now set the display resolution to 1280*1024 and browser zoom to 400% and check whether the horizontal scroll bar appears or not.

**Actual:**
Horizontal scroll bar appears when the page is zoomed to 400%.

**Expected:**
Zooming up to 400% must not trigger two-dimensional scrolling (both horizontal and vertical) or create a loss of content or functionality.

**User Impact:**
People with visual disabilities must be able to scale the browser up to 400% without losing the ability to read and track text. Two-dimensional scrolling impairs tracking and reading text, thus blocking the ability to understand or use the content. 
Note: Same issue is observed in ‘Playground’ page.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/reflow

https://user-images.githubusercontent.com/88436727/150644444-cf769fc4-99e6-4c15-8897-38b1c92b79ee.mp4

**Comment by octogonz** [comment]
This issue has been closed because this Jekyll project has been retired. The documentation content has been migrated to Docusaurus. 

If the problem has not been addressed, please file a new issue under this GitHub project instead:  

https://github.com/microsoft/rushstack-websites
